client/question/services/EventHandlerService.js

```diff
@@ -19,7 +19,38 @@
 tie.factory('EventHandlerService', [
   '$http', 'ServerHandlerService', function($http, ServerHandlerService) {
 
+
+    /**
+     * Global object to keep track of the current batch of events to send.
+     * @type [Event]
+     */
+    var currentEventBatch = [];
+
+    /**
+     * Submits the current batch of event data to TIE's backend.
+     *
+     */
+    var sendCurrentEventBatch = function() {
+      var data = {
+        events: currentEventBatch,
+        createdMsec: (new Date()).getTime()
```

Comment:
I thought this was going to be timeSentToBackendMsec?

Reply:
Oop; you're right, sorry.